advocacy_docs/edb-postgres-ai/ai-accelerator/models/supported-models/bert.mdx

```diff
@@ -51,11 +51,11 @@ select aidb.create_model(
 
 ## Model configuration settings
 
-The following configuration settings are available for CLIP models:
+The following configuration settings are available for BERT models:
```

Comment:
Please check for similar copy/paste errors (assuming this is one) throughout this section. I couldn't tell if the others were intentional mismatches with what appeared to be the model name.